src/components/augmentedReality/AugmentedRealityView.js

```diff
@@ -76,41 +71,78 @@ export const AugmentedRealityView = ({ sceneNavigator }) => {
   );
 };
 
+// eslint-disable-next-line complexity
 const ViroSoundAnd3DObject = (item) => {
   const {
     isObjectLoading,
     isStartAnimationAndSound,
     object,
-    position,
-    rotation,
-    scale,
     setIsObjectLoading,
     setIsStartAnimationAndSound
   } = item;
 
+  // if the `chromaKeyFilteredVideo` prop is undefined, the default colour of GreenScreen is defined
+  ViroMaterials.createMaterials({
+    chromaKeyFilteredVideo: {
+      chromaKeyFilteringColor: object?.mp4?.chromaKeyFilteredVideo || '#00FF00'
+    }
+  });
```

Comment:
is this ok to be here without a condition? is this necessary in any case? or would it make sense to have a condition to add that materials just in case we have video stuff?

Reply:
Yes, you're right. 
https://github.com/ikuseiGmbH/smart-village-app-app/pull/634/commits/c69f385584050e19f46a35da65fddf329cc59a5c